Makefile

```diff
@@ -56,6 +56,7 @@ SOURCE_FILE_BASE_URL="https://sonicstorage.blob.core.windows.net/debian-security
 DSC_FILE_URL = "$(SOURCE_FILE_BASE_URL)/$(DSC_FILE)"
 DEBIAN_FILE_URL = "$(SOURCE_FILE_BASE_URL)/$(DEBIAN_FILE)"
 ORIG_FILE_URL = "$(SOURCE_FILE_BASE_URL)/$(ORIG_FILE)"
+NON_UP_LOC = non_upstream_patches
```

Comment:
Can you rename this to `NON_UP_DIR`, to make it clear it's a directory? Also, this directory should be outside of the kernel source code, so that there's no chance of this directory conflicting with anything coming from the kernel (or Debian) itself.

Reply:
Handled